src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -81,11 +81,23 @@
      * @param reason
      * @return Accession with complete patch information
      * @throws AccessionDoesNotExistException when the accession has never existed.
-     * @throws HashAlreadyExistsException     when another accessioned object exists already with the same hash
      * @throws AccessionDeprecatedException   when the accession exists but has been deprecated
      * @throws AccessionMergedException       when the accession exists but has been merged into another accession
      */
     void deprecate(ACCESSION accession, String reason) throws AccessionMergedException, AccessionDoesNotExistException,
             AccessionDeprecatedException;
 
+    /**
+     * Merges an accession into another
+     *
+     * @param accessionOrigin
+     * @param accessionDestiny
+     * @param reason
+     * @throws AccessionDoesNotExistException when either accession has never existed.
+     * @throws AccessionDeprecatedException   when either accession exists but has been deprecated
+     * @throws AccessionMergedException       when either accession exists but has been merged into another accession
+     */
+    void merge(ACCESSION accessionOrigin, ACCESSION accessionDestiny, String reason) throws AccessionMergedException,
```

Comment:
`destination` instead of `destiny` (another raid? :stuck_out_tongue: )

Affects several classes.